.github/workflows/mavencentral-publish.yml

```diff
@@ -39,8 +39,14 @@ jobs:
       - name: Release build
         run: |
           cd thirdparty/godot-cpp
-          scons platform=android target=template_debug -j4
-          scons platform=android target=template_release -j4
+          scons platform=android target=template_debug arch=arm64 -j4
```

Comment:
I forgot to remove the `-j4` here. Scons now autodetects this variable when not specified. So if more cores are available it will use them.